http-gateway/web/src/components/menu/menu-item.js

```diff
@@ -12,7 +12,12 @@ export const MenuItem = ({
   to,
   ...rest
 }) => {
-  const menuItemClassName = classNames('menu-item', className)
+  const location = useLocation()
+  const menuItemClassName = classNames(
+    'menu-item',
+    className,
+    to === '/' && location.pathname.includes('devices') ? 'active' : ''
```

Comment:
I agree -> It can be set as a prop. 
Why I need this -> It comes from concept: 

Menu with link "to" is not active only for "/" but also for "/devices/X/Y. So active is not only for home route but also for details of devices.